tests/contrib/mysqldb/test_mysqldb.py

```diff
@@ -614,3 +617,24 @@ def test_user_specified_service_integration(self):
         spans = tracer.pop()
 
         assert spans[0].service == "mysvc"
+
+    def test_trace_connect(self):
+        self._add_dummy_tracer_to_pinned(MySQLdb)
+
+        # No span when trace_connect is False (the default)
+        self._connect_with_kwargs().close()
+        spans = self.tracer.pop()
+        assert not spans
+
+        with self.override_config("mysqldb", dict(trace_connect=True)):
```

Comment:
Worth testing the environment variable too (see the `test_user_specified_service_integration` above for a good way to do this)